src/fmeval/transforms/summarization_accuracy_metrics.py

```diff
@@ -36,67 +37,86 @@ class SummarizationAccuracyMetric(Transform):
 
     def __init__(
         self,
-        target_output_keys: List[str],
-        model_output_keys: List[str],
         output_keys: List[str],
+        model_output_keys: List[str],
         allow_duplicate_input_keys: bool,
+        target_output_keys: Optional[List[str]] = None,
+        target_output_keys_provider: str = "",
```

Comment:
We can use `Optional[str]` here to avoid using an empty string as the default value.